firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -232,17 +237,35 @@ public UpdateTask updateApp() {
    * Overloaded updateApp with boolean input showDownloadInNotificationsManager. Set to true for
    * basic configuration and false for advanced configuration.
    */
-  private UpdateTask updateApp(boolean showDownloadInNotificationManager) {
+  public UpdateTask updateApp(boolean showDownloadInNotificationManager) {
```

Comment:
Just wanted to call out that this is public now. Since updateApp was public as a class and now that functionality is here it makes sense to me to make it public. Thoughts?

Reply:
this should still be private. This method overload with the boolean parameter should only be called by our SDK. External packages should call the overload above with no parameters.

I don't think UpdateApp should have been a public class. That was an oversight on our part. The public/private audit going forward will probably make most classes package private

Reply:
That makes sense. Will change back.